src/main/scala/org/coroutines/Coroutine.scala

```diff
@@ -103,6 +103,10 @@ object Coroutine {
     var $yield: Y = null.asInstanceOf[Y]
     var $result: R = null.asInstanceOf[R]
 
+    /** Clones the coroutine that this instance is a part of.
```

Comment:
Thanks for following up!

Can you please make sure that the indentation in the doc comments exactly matches the one specified in the doc I linked (`*`)?
At the moment, our linter does not verify these rules, but we'll ensure that it that in the future.